x-pack/plugins/ml/public/application/components/annotations/annotations_table/annotations_table.js

```diff
@@ -67,13 +67,18 @@ export class AnnotationsTable extends Component {
     this.state = {
       annotations: [],
       isLoading: false,
+      queryText: `event:(${ANNOTATION_EVENT_USER})`,
```

Comment:
The default query should also include `event:delayed_data` as well as `user`.

![image](https://user-images.githubusercontent.com/7405507/87059333-17e71f00-c201-11ea-8f45-ec05da7c3e15.png)